ale_linters/c/cquery.vim

```diff
@@ -6,6 +6,9 @@ call ale#Set('c_cquery_cache_directory', expand('~/.cache/cquery'))
 
 function! ale_linters#c#cquery#GetProjectRoot(buffer) abort
     let l:project_root = ale#path#FindNearestFile(a:buffer, 'compile_commands.json')
+    if empty(l:project_root)
+        let l:project_root = ale#path#FindNearestFile(a:buffer, '.cquery')
+    endif
     return !empty(l:project_root) ? fnamemodify(l:project_root, ':h') : ''
```

Comment:
Put a blank line before every`return` that doesn't start a new level of indentation.